classes/module/Module.php

```diff
@@ -1691,6 +1691,18 @@ public static function getModulesInstalled($position = 0)
         return Db::getInstance()->executeS($sql);
     }
 
+    /**
+     * Return modules that are installed AND enabled
+     *
+     * @return array Modules
+     */
+    public static function getModulesInstalledAndEnabled()
```

Comment:
I thought "mmmm why not add a parameter `$isDisabled` to `getModulesInstalled()` but then I remembered that the less arguments there is the better it is. So OK for me 👍

Reply:
as seen with @atomiix , this method is not multishop compatible (modules are actually activated per shop now)

Reply:
I'm updating the code